src/test/kotlin/blackjack/domain/CardTest.kt

```diff
@@ -0,0 +1,33 @@
+package blackjack.domain
+
+import blackjack.model.Card
+import blackjack.model.CardShape
+import blackjack.model.CardType
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.Arguments
+import org.junit.jupiter.params.provider.MethodSource
+import java.util.stream.Stream
+
+internal class CardTest {
+
+    @DisplayName("각 카드 타입 별로 유효한 값(value)과 점수 및 모양을 가지고 있다.")
+    @ParameterizedTest
+    @MethodSource("provideValidCard")
+    fun validate(card: Card, value: String, score: Int, shape: String) {
```

Comment:
```suggestion
    @ParameterizedTest
    @MethodSource("provideValidCard")
    fun `각 카드 타입 별로 유효한 값과 점수 및 모양을 가지고 있다`(card: Card, value: String, score: Int, shape: String) {
```
JUnit5에서는 다음처럼 함수명으로 한글을 작성할 수 있습니다 👍  가독성을 높이기에 좋아요.